galter_subjects_utils/contrib/mesh/cli.py

```diff
@@ -185,6 +186,12 @@ def mesh_deltas(**parameters):
         )
         .convert()
     )
+    # in-place
+    KeepTrace.mark(
+        ops,
+        # only can keep trace of *those* anyway
+        yes_logic=lambda op: op["type"] in ["rename", "replace", "remove"]
```

Comment:
Ah, I see, you're passing the lambda for use in KeepTrace. That's pretty cool 😎 

Are you anticipating this changing or do you just like this pattern? I think it's a good approach regardless, I just tend to go towards more naive approaches and would probably put the yes_logic in the mark function itself.

Reply:
Reflexively I was trying to isolate the decision logic. Then I was thinking of providing flexibility to mark only deltas for subjects we use versus for subjects that are removed/replaced/renamed, but realized that didn't matter since only subjects we used would leave a trace (duh). But then, I used that flexibility for little scripts to generate sample `deltas.csv` files for manual testing which is not a major advantage either. In the end, providing the logic in `mark` would have been equally fine too. I'll keep an eye on how that develops in the future.